eventbrite/eventbrite-connector/eventbrite-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/eventbrite/EventbriteConnectorIntegrationTest.java

```diff
@@ -51,10 +52,13 @@ public void setEnvironment() throws Exception {
         super.init();
         ConfigurationContextProvider configurationContextProvider = ConfigurationContextProvider.getInstance();
         ConfigurationContext cc = configurationContextProvider.getConfigurationContext();
-        mediationLibUploadStub = new MediationLibraryUploaderStub(cc, esbServer.getBackEndUrl() + "MediationLibraryUploader");
+        mediationLibUploadStub =
+                new MediationLibraryUploaderStub(cc, esbServer.getBackEndUrl() + "MediationLibraryUploader");
         AuthenticateStub.authenticateStub("admin", "admin", mediationLibUploadStub);
```

Comment:
Why is the credentials hardcoded here? What happens when we change the admin password from the default one?